google-cloud-pubsub/samples/topics.rb

```diff
@@ -290,6 +290,37 @@ def publish_messages_async_with_concurrency_control topic_id:
   # [END pubsub_publisher_concurrency_control]
 end
 
+def publish_messages_async_with_flow_control topic_id:
+  # [START pubsub_publisher_flow_control]
+  # topic_id = "your-topic-id"
+  require "google/cloud/pubsub"
+
+  pubsub = Google::Cloud::Pubsub.new
+
+  topic = pubsub.topic topic_id, async: {
+    # Configure how many messages the publisher client can hold in memory
+    # before publishing succeeds, and what to do when messages exceed the limit.
```

Comment:
What do you mean by "before publishing succeeds?"

Reply:
Removed, thanks!